src/main/webapp/aboutus.html

```diff
@@ -66,9 +66,9 @@ <h2>Brandy Huang</h2>
         dream in my childhood and it sounds really cool to flying in the endless sky.
     </li>
     <li>Ask me About:<a href="https://movieweb.com/minions-2-the-rise-of-gru-release-date-summer-2020/">Minions 2: The
-        Rise of Gru Is Officially Coming in Summer 2020 !!</a> Oh yeah!! I love watching Minions series movies.They are
-        so cute and funny. I have blanket, dolls on my bed and bags are all Minions. Can't wait to see the movie!!
-    </li>
+    Rise of Gru Is Officially Coming in Summer 2020 !!</a> Oh yeah!! I love watching Minions series movies.They are
```

Comment:
Not sure if this was a merge issue, but why did you move this outside of the <li> tag?

Reply:
I use sublime text at first and reformat it on this platform. So I guess that's why I change other people's code unconsciously:( I change to Intellij now and I know the way they reformat code is different haha